src/components/CatalogOverlay/CatalogPlotComponent/CatalogPlotComponent.tsx

```diff
@@ -287,6 +281,15 @@ export class CatalogPlotComponent extends React.Component<WidgetProps> {
         return profileInfo;
     }
 
+    private initnBinx(): number {
```

Comment:
I think we should rename this to `numBinsX` so that we have `initNumBinsX` and `setNumBinsX`
`initnBinx` doesn't adhere to our naming conventions.